exercises/F-predicates/exercise.js

```diff
@@ -12,6 +12,7 @@ function isNegative(number) {
 }
 
 // Finish the predicate function to test if the passed number is between 0 and 10
+
 function isBetweenZeroAnd10(number) {}
 function isBetweenZeroAnd10(number) {
   return number > 0 && number < 10;
```

Comment:
Do you think, 0 and 10 should be inclusive in the function? I think there is a missing closing curly brace for the function. And there appears to be double function declaration, so you could delete the first.